samples/erc20-sample/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+
+ARG GO_VER=1.17.5
+ARG ALPINE_VER=3.14
+
+FROM  172.22.50.223/bestchains-dev/golang:${GO_VER}-alpine${ALPINE_VER} as build
```

Comment:
should use `FROM golang:${GO_VER}-alpine${ALPINE_VER} as build` instead of `172.22.50.223/bestchains-dev/golang`